universal-application-tool-0.0.1/tailwindbuilder.sbt

```diff
@@ -0,0 +1,17 @@
+import scala.sys.process.Process
+
+lazy val tailwindCli = TaskKey[Unit]("run webpack when packaging the application")
+
+def runTailwindCli(file: File) = {
+  Process("npx tailwindcss-cli@latest build ./app/assets/stylesheets/styles.css -o ./public/stylesheets/tailwind.css", file) !
```

Comment:
suggestion (non-blocking): It might be nice to define this command string in a constant somewhere so it can be shared with the one in `Tailwindbuilder.scala`, especially since both places would need to be updated if we ever made a change to this command. But I have no idea how dependencies between `.scala` and `.sbt` files work, so no worries if it's a pain.

Reply:
I will publish further documents on this and add the constant as suggested.